tools/static_check_containers

```diff
@@ -8,5 +8,5 @@ if [ "$(curl -s -w "%{http_code}" https://ghcr.io/v2/ -o /dev/null)" = "401" ];
 else
     img=ghcr.io/hadolint/hadolint
 fi
-cre="${cre:-"docker"}"
+cre="${cre:-"podman"}"
```

Comment:
Should there be a change to remove or disable the docker daemon?

Reply:
I don't think we explicitly install docker anywhere. Let's see what happens if I replace more. Updated the PR to include a second commit which generalizes more places.